Resources/Prototypes/Datasets/Names/regalrat.yml

```diff
@@ -0,0 +1,39 @@
+- type: dataset
+  id: RegalRatNameKingdom
+  values:
+  - Plague
+  - Miasma
+  - Maintenance
+  - Trash
+  - Garbage
+  - Rat
+  - Vermin
+  - Cheese
+  - Munching
+  - Big
+  - Filth
+  - Sewer
+  - Disposal
```

Comment:
Maybe "Disposals"? Up to you.

Reply:
"disposals lord" sounds worse than "disposal lord" imo.

Reply:
Yeah, sure.